### DISCUSSION TITLE
Layered navigation now working with smile/elasticsuite module

### DISCUSSION THREAD

**Discussion by webcreative24** [breezefront/community#37] [3. Integrations]
Hello. For our stores, we use a very useful module https://github.com/Smile-SA/elasticsuite, but after installing the Breeze theme, the filters are not displayed in the categories, which is very sad and it would be cool to be able to use this cool module and your cool Breeze theme. Can you help, please? Thanks

**Comment by dimdimich** [comment]
Our developers will check if we they can work on integration list but we can't guarantee that for now.

**Reply by webcreative24** [reply]
Thanks

**Comment by vovayatsyuk** [comment]
Hi,

Can you please investigate it a bit more? Eg. do you see any fatal errors in your logs?

Can you also check if this line returns true in elasticsuite template: https://github.com/Smile-SA/elasticsuite/blob/2.10.x/src/module-elasticsuite-catalog/view/frontend/templates/layer/view.phtml#L22

**Reply by webcreative24** [reply]
Hi. There are no errors in the logs And I can confirm that https://github.com/Smile-SA/elasticsuite/blob/2.10.x/src/module-elasticsuite-catalog/view/frontend/templates/layer/view.phtml#L22 returned true

Also, when turning off breeze mode via url (?breeze=0) then I can see layered nav

**Reply by vovayatsyuk** [reply]
Since it returns true I expect that the HTML from that block should be rendered and you should see something (error, or partially rendered block). Can you please check it?

**Reply by webcreative24** [reply]
unfortunately, there are no errors

**Reply by webcreative24** [reply]
Hello. @vovayatsyuk 
I just finding out that all filters are present on the nav, but they are not visible
http://joxi.ru/gmv0wwPTekMaNA
I think it's related to accordion. Are there some changes or replacement in the Breeze theme for the accordion?
